source/jormungandr/tests/chaos_disruptions_tests.py

```diff
@@ -640,17 +640,207 @@ def test_disruption_on_journey(self):
         We delete the network disruption and test if there's a journey using
         the line B.
         """
-        response = self.query_region(journey_basic_query)
 
+        def contains_pt_journey(resp):
+            return any(
+                [
+                    len([s for s in j["sections"] if s["type"] == "public_transport"]) == 1
```

Comment:
```suggestion
                    len([s for s in j["sections"] if s["type"] == "public_transport"]) >= 1
```
otherwise this would return true if only if there is a journey which with exactly one section of public_transport.
Given the name of the function, I expect it to return true if resp contains a journey with 2 sections of public_transport